### PR TITLE
Upgrade to Oracle GraalVM for JDK 21

### PR DESCRIPTION
**Pull request by olyagpl**

### CODE REVIEW

**Comment by olyagpl** [comment]
I had made this demo look and run using the same settings and versions as Jibber demo used in this Luna Lab  https://luna.oracle.com/lab/268ea851-2f09-43e6-8d70-40a10cb4de03. GitHub Action workflow passed.

**Review by LesiaChaban — APPROVED** [review]

**Comment by sachin-pikle** [comment]
Made some updates to the readme and pom.xml.

**Review by sachin-pikle — APPROVED** [review]
Looks good to me.

File: spring-native-image/Dockerfiles/Dockerfile.jvm [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Using Oracle GraalVM for JDK 17
```
Comment (gustavo-flor) [inline spring-native-image/Dockerfiles/Dockerfile.jvm:1]: @sachin-pikle maybe we should update the comment too?

Comment (sachin-pikle) [reply spring-native-image/Dockerfiles/Dockerfile.jvm:1]: Yes. Will fix it in the next update.

**Pull request merged**